office365_admin/requirements.txt

```diff
@@ -1,4 +1,4 @@
 # List third-party dependencies here, separated by newlines. 
 # All dependencies must be version-pinned, eg. requests==1.2.0
 # See: https://pip.pypa.io/en/stable/user_guide/#requirements-files
-requests==2.18.4
\ No newline at end of file
+requests==2.20.0
```

Comment:
requests in included in the base plugin image. We should not have it here as this will override the base requests and make keeping requests updated harder than it needs to be

Reply:
tested, yes you are right.